config/locales/es_ar.yml

```diff
@@ -1,4 +1,4 @@
-es_ar:
+es-AR:
```

Comment:
There's something wrong with Crowdin's integration that's converting all locale root keys into an incorrect format. This must be fixed before we can merge more of these.